server/src/main/java/org/elasticsearch/action/ingest/GetPipelineRequestBuilder.java

```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.master.MasterNodeReadOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class GetPipelineRequestBuilder extends MasterNodeReadOperationRequestBuilder<GetPipelineRequest, GetPipelineResponse, GetPipelineRequestBuilder> {
+public class GetPipelineRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<GetPipelineRequest, GetPipelineResponse, GetPipelineRequestBuilder> {
```

Comment:
Could you indent this one more time so the class declaration doesn't line up with the class body?

Reply:
It is my pet complaint to IntelliJ users :smile: